core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java

```diff
@@ -140,6 +135,11 @@ public String getName() {
      */
     public abstract Type type();
 
+    @Override
+    public String getWriteableName() {
+        return type().stream().toUtf8();
```

Comment:
ok if it stays as it used to be, leave it! I was afraid it was a bug introduced here